app/portainer/components/datatables/genericDatatableController.js

```diff
@@ -62,10 +114,11 @@ function (PaginationService, DatatableService) {
       this.state.textFilter = textFilter;
       this.onTextFilterChange();
     }
-  };
 
-  function setDefaults(ctrl) {
-    ctrl.showTextFilter = ctrl.showTextFilter ? ctrl.showTextFilter : false;
-    ctrl.state.reverseOrder = ctrl.reverseOrder ? ctrl.reverseOrder : false;
-  }
+    var storedFilters = DatatableService.getDataTableFilters(this.tableKey);
+    if (storedFilters !== null) {
+      this.filters = storedFilters;
+    }
+    this.filters.usage.open = false;
```

Comment:
2 things to do here:
1. protect the line to avoid assignation on `undefined` `filters` or `usage`
2. (optionnal): refactor `ImageDatatable` and `VolumesDatatable` to rename `this.filters.usage` to `this.filters.state` (need also to be refactored in HTML files)

Step 2 is only pointed to keep the code consistent across DatatableControllers, but can be done later.

Reply:
@xAt0mZ done